package.json

```
@@ -19,7 +19,10 @@
     "lint": "gulp lint",
     "build": "gulp build",
     "dist": "gulp dist",
-    "heroku-postbuild": "gulp clean && gulp build && gulp dist --fortesting"
+    "heroku-postbuild": "gulp clean && gulp build --fortesting && gulp dist --fortesting && npm run try-prepend",
+    "try-prepend": "if [ -n \"$AMP_TESTING_HOST\" ]; then npm run prepend-amp; npm run prepend-v0; fi",
```

Can we just fail if this is not present with a message what to do?